confgenerator/logging_processors.go

```diff
@@ -523,3 +523,28 @@ func (p LoggingProcessorExcludeLogs) Processors(ctx context.Context) ([]otel.Com
 func init() {
 	LoggingProcessorTypes.RegisterType(func() LoggingProcessor { return &LoggingProcessorExcludeLogs{} })
 }
+
+// LoggingMultiProcessorMixin implements the methods required to describe a pipeline with one or more log processors.
+type LoggingMultiProcessorMixin interface {
```

Comment:
`LoggingMultiProcessorMixin` -> `LoggingProcessorMacro`, the idea being that it's a C-style macro to transform Ops Agent processor config into Ops Agent processor(s) config

Reply:
Done